app/controllers/PlutoController.scala

```diff
@@ -31,4 +43,34 @@ class PlutoController(
       Ok(Json.toJson(project))
     }}
   }
+
+  def resendAtomMessage(id: String) = APIHMACAuthAction {
+    val settings = com.gu.media.Settings(this.config)
+    val pluto = new PlutoUploadActions(awsConfig)
```

Comment:
we can create this variable a bit later within the `atom.plutoData` pattern match

Reply:
Sure. Out of interest, why does this matter?

Reply:
clarity really - there's no point in instantiating it if it won't be used, which is true if there is no `plutoData`